scheduler/cloudformation_templates/essentials_encrypted_ebs_kms_policy_all.yml

```diff
@@ -0,0 +1,31 @@
+Description:
+  Allow Essentials Lambda Functions to use KMS for starting EC2s with encrypted EBS
+
+Parameters:
+    lambdaExecutionRole:
+        Description: The execution role name for Essentials Lambda. Please change it if you use different stack and role name.
+        Type: String
+
+Resources:
+  KMSEBSAccessPolicy:
+    Type: 'AWS::IAM::ManagedPolicy'
+    Properties:
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: AllowEssentialsLambdaToUseEncryptedEBS 
+            Effect: Allow
+            Action:
+              - 'kms:RetireGrant'
+              - 'kms:CreateGrant'
+              - 'kms:ReEncrypt*'
+              - 'kms:GenerateDataKey*'
+              - 'kms:Encrypt'
```

Comment:
do we need to use any wildcard here?

Reply:
Yes, that's what I'm offering with this. Copilot already does it. I'm thinking we can offer two buttons and both options let them decide if they want to do one-byy-one with each ARN or they want all in one go.